Improvements for Cloud Docker topic (#4512)
magento/devdocs: guides/v2.1/cloud/docker/docker-config.md

127.0.0.1 magento2.docker
```

You can also run the command in your CLI:

```bash
echo "127.0.0.1 magento2.docker" | sudo tee -a /etc/hosts
```

### Stopping default Apache instance on Mac OS

> Because Mac OS provides built-in Apache service, and may occupy port `80` you'll have to stop it with:

Reviewer: ```suggestion
Because Mac OS provides built-in Apache service, and may occupy port `80`, you must stop the service with the following command:
```

```bash
docker-sync stop
```

## Advanced usage

### Extending docker-compose.yml configuration

> You have a possibility to use Docker's built-in [extension mechanism](https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files).

Reviewer: ```suggestion
You can use Docker's built-in [extension mechanism](https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files).
```

1.  Create a `docker-compose-dev.yml` file inside your project's root directory with next content:

```yaml
version: '2'
services:
  deploy:
    environment:
      - ENABLE_SENDMAIL=true
```

> This will replace default value of `ENABLE_SENDMAIL` environment variable.

Reviewer: ```suggestion
    This replaces the default value of the `ENABLE_SENDMAIL` environment variable.
```
